centaur/src/main/resources/standardTestCases/docker_hash/docker_hash_dockerhub_private_google_secret_manager.wdl

```diff
@@ -0,0 +1,13 @@
+task dockerhub {
+    command {
+        echo "hello"
+    }
+    runtime {
+        docker: "broadinstitute/cloud-cromwell:dev"
```

Comment:
This image is so old it does not pull or run in Docker Desktop. Obviously not the only place we use it, but it might be worth not propagating it into the new backend.

```
> docker run -it --entrypoint /bin/bash broadinstitute/cloud-cromwell:dev
Unable to find image 'broadinstitute/cloud-cromwell:dev' locally
dev: Pulling from broadinstitute/cloud-cromwell
docker: [DEPRECATION NOTICE] Docker Image Format v1 and Docker Image manifest version 2, schema 1 support is disabled by default and will be removed in an upcoming release. Suggest the author of docker.io/broadinstitute/cloud-cromwell:dev to upgrade the image to the OCI Format or Docker Image manifest v2, schema 2. More information at https://docs.docker.com/go/deprecated-image-specs/.
```

Reply:
This image does pull and run on both PAPI v2 and Batch, but I'll see if I can push something a little less archaic.

Reply:
It looks like you pushed updates to it in August, seems like we might just change the tag

https://hub.docker.com/repository/docker/broadinstitute/cloud-cromwell/general

Reply:
updated to reference one of those August images 🤞